backends/backend.conf

```diff
@@ -4,6 +4,104 @@ backend {
   default = "Local"
   providers {
 
+    slurm_singularity {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        script-epilogue = "sleep 30"
+        concurrent-job-limit = 50
+        runtime-attributes = """
+          Int cpu = 1
+          Int? time
+          Int? memory_mb
+          String? slurm_partition
+          String? slurm_account
+          String singularity_container
+          String? singularity_options
+          String singularity_command = "exec"
+          String? singularity_command_options
+        """
+        submit = """
+          sbatch \
+          --export=ALL \
+          -J ${job_name} \
+          -D ${cwd} \
+          -o ${out} \
+          -e ${err} \
+          ${"-t " + time*60} \
+          -n 1 \
```

Comment:
This shouldn't be controllable?

Reply:
`-n 1` specifically

Reply:
I am not sure if our pipelines need to use multiple nodes.

`-n` defines number of nodes and we fix it as 1. The actual number of cpus (`$cpu`) for a job is defined by `--ntasks-per-node=1 --cpus-per-task=$cpu`.

Reply:
If that's the case, then the hard coding is A-OK! If there were a more general template (that we wanted to give the user choice) I would suggest this to be a variable. If the pipeline could break or otherwise not function correctly with more than 1, then probably better to have it set like this!